3529/1/apihandler.rb

```diff
@@ -0,0 +1,30 @@
+require 'json'
+require 'open-uri'
+require 'httparty'
+
+class GemsApiHandler
+  attr_reader :gem_github
+
+  def initialize(gem_name)
+    @gem_name = gem_name
+    @url = HTTParty.get("https://rubygems.org/api/v1/gems/#{@gem_name}.json")
+  end
+
+  def find_github
```

Comment:
Find github what? Website? Funny Octocat? Please, rename method name for better understanding what it does